ol7/profiles/pci-dss.profile

```diff
@@ -0,0 +1,111 @@
+documentation_complete: true
+
+title: 'PCI-DSS v3 Control Baseline Draft for Oracle Linux 7'
+
+description: 'Ensures PCI-DSS v3 related security configuration settings are applied.'
+
+selections:
```

Comment:
I don't see the benefit of having those commented lines here.
I guess that you want to enable those rules later, but having those lines doesn't have any benefit. Anybody can use tools based on `diff` to see the difference between the `ol7` and e.g. `rhel7` PCI-DSS profile to see what rules are missing. The git-centric point of view when you add a commented line, which you later substitute for line without comment doesn't make sense either.

Reply:
Far easier to comment out rules that need to be enabled later than create 100+ github tickets to track things. I'm guilty of using comments as placeholders too. 

Yes, tools like ``diff`` could be used to see differences vs github tickets. Each developer has their own workflow -- for myself, having to use ``diff`` and figure out how to parse the results would be incredibly annoying. Especially when trying to update 100+ rules.... running diff 100+ times? No way.

In the pursuit of project usability/accessibility/ease of contributing vs elegant git logs, ease of contributing should be favored. Suggest we nudge PRs to use "clean" commits as @matejak suggested but not block PRs over the issue.

Reply:
Guys, thanks for your comments.
I just considered having entire list of rules would be easier to track and maintain. However no problem with removing commented out rules as I definitely can keep this list outside of source project.
Could you please have a look at the updated request?
Note: Due to number of origins on my system incorrect commit had been pushed by mistake first. It is updated, rebased and hope fine now.